### PR TITLE
Adds the preview/deploy steps to the readme.

### PR DESCRIPTION
**Pull request by EvanYoung-SP**
### What changed
This PR adds some steps on how to use and implement previews on the gatsby support doc site.

### Steps to Test
The preview process:

- create a branch
- create a PR to master
- go to https://www.gatsbyjs.com/dashboard/organization/60c30363-21fd-4739-a2ca-75f5011a3a1b/sites
- click on the link "Support Docs"
- you are now in the "builds" section. here you can see a "production" section and a "pull requests" section
- look in the "pull requests" section for the PR you just created and click "view build"
- you should see the changes you made
- now, go back to the build section and in the "production" section, click on the "view build" for the only entry there
- you shouldn't see your changes as expected since this is prod

The build process:

- if you want to verify this, just merge your PR into master and observe the build in the gatsbyjs dashboard mentioned above
- once the build finishes, you can see the build occurring in the "production" section.
- once it's complete, you can then view the build or go to https://supportdocs.gatsbyjs.io/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,11 +217,29 @@ If you want to clean out the navigation, articles, categories, and related media
...
-
 ### GATSBY!! (or the start of it)
+
 Gotta start somewhere! I'm putting a placeholder header section for now that will further outline how to work with support-docs on Gatsby.
 
+#### How to preview changes
+
+1. Create a branch
+2. Create a PR to master
+3. Go to https://www.gatsbyjs.com/dashboard/organization/60c30363-21fd-4739-a2ca-75f5011a3a1b/sites
```
Comment (maxharrisnet) [inline README.md:228]: You might want to mention something about needing an account. Something like "contact the FE team if you need an account".

File: README.md [hunk 2]
```diff
@@ -221,11 +217,29 @@ If you want to clean out the navigation, articles, categories, and related media
...
+4. Click on the link "Support Docs"
+5. You are now in the "builds" section. here you can see a "production" section and a "pull requests" section
```
Comment (maxharrisnet) [inline README.md:230]: Nitpick: "Here" should be capitalized.

File: README.md [hunk 3]
```diff
@@ -221,11 +217,29 @@ If you want to clean out the navigation, articles, categories, and related media
...
+6. Look in the "pull requests" section for the PR you just created and click "view build"
+
+- This will show a version for the PR current build of the static gatsby site
+
+#### How to deploy changes
+
+1. Complete the steps above in the "How to preview changes" section
+2. Once the PR is approved by a reviewer and the PR is merged and squashed, the build will kick off automatically.
```
Comment (maxharrisnet) [inline README.md:238]: Do we need to specify the branch for production builds?

File: README.md [hunk 4]
```diff
@@ -221,11 +217,29 @@ If you want to clean out the navigation, articles, categories, and related media
...
+
 #### Uptime monitoring
+
 We plan on leveraging CloudWatch Canary with Terraform, but right now we have access to Gatsby's uptime monitoring on their site.
+
```
Comment (maxharrisnet) [inline README.md:243]: It might be worth mentioning Lighthouse reports in Gatsby Cloud as well.

File: README.md (hunk 2 above)
Comment (EvanYoung-SP) [reply README.md:230]: Nitpick away! Good catch!

File: README.md (hunk 3 above)
Comment (EvanYoung-SP) [reply README.md:238]: I personally don't think so since merges to master/main will kick off the build and that's just git knowledge.

**Review by maxharrisnet — APPROVED** [review]

**Pull request merged**